src/server/endpoints/covidcast.py

```diff
@@ -144,6 +145,15 @@ def handle():
         q.set_order("source", "signal", "time_type", "time_value", "geo_type", "geo_value", "issue")
     q.set_fields(fields_string, fields_int, fields_float)
 
+    missing_fields = ["missing_value", "missing_stderr", "missing_sample_size"]
+    fields_int.extend(missing_fields)
+    if sql_table_has_columns("covidcast", missing_fields):
+        # real fields
+        q.fields.extend([f"{q.alias}.{field}" for field in missing_fields])
+    else:
+        # fake fields
+        q.fields.extend([f"0 as {field}" for field in missing_fields])
```

Comment:
Are we ok with setting 0 as the default value (as opposed to "not implemented" or something), since it means entry not missing, which isn't necessarily true for existing data.

Reply:
what would be an alternative value?

Reply:
That's....a good question. Does it make sense to not return those fields at all?

Reply:
Prefer not returning those fields at all, unless that will continue to block a project we were hoping to unblock with this PR. How many of the things that are currently blocked on nans, actually need the missingness columns and not just a compatible database+server?

- [ ] dashboard builder
- [ ] speed improvements for frontend queries (there are piles of these; are any of them trying to explicitly exclude the missingness columns? & liable to get confused when there are no missingness columns to exclude?)
- [ ] advanced metadata
- [x] quidel revisions (need missingness annotations to handle deletions)

Reply:
for now I changed it that the columns are not returned. I guess for the quidel revisions returning dummy values won't help this sub project either